tests/rptest/services/redpanda.py

```diff
@@ -2476,6 +2476,11 @@ def __init__(self,
 
         if node_ready_timeout_s is None:
             node_ready_timeout_s = RedpandaService.DEFAULT_NODE_READY_TIMEOUT_SEC
+
+        # Adjust timeout if cloud provider is Azure by adding 10 seconds to the default
+        if self._context.globals.get("cloud_provider") == "azure":
```

Comment:
nit: any issues bumping for all providers?

Reply:
We could.. But it works for other provides. Only on Azure it requires more time.. I didn't want to increase timeout for other providers, so we could catch changes in case it starts to fail.. We _could_ bump for all of them

Reply:
I guess I have a slight preference for doing it for all providers. H/e if we do it for Azure only we should provide a better comment, linking back to the issue or somewhere else with a good discussion of why.

Reply:
Sounds good, let's update it for all providers. I'll push another commit